src/index.jsx

```diff
@@ -53,10 +54,16 @@ const App = () => {
           <Route path="/course/:courseId/*" element={<CourseAuthoringRoutes />} />
           <Route path="/course_rerun/:courseId" element={<CourseRerun />} />
           {process.env.ENABLE_TAGGING_TAXONOMY_PAGES === 'true' && (
-            <Route
-              path="/taxonomy-list"
-              element={<TaxonomyListPage />}
-            />
+            <>
+              <Route
+                path="/taxonomy-list"
+                element={<TaxonomyListPage />}
+              />
+              <Route
+                path="/tagging/components/widget/:unitId"
+                element={<UnitTaxonomyTagsDrawer />}
```

Comment:
In the (very near) future, this will be used for components too (not just units), so maybe `ContentTagsDrawer` would be a better name.